pkg/scheduler/scheduler.go

```diff
@@ -66,7 +73,7 @@ type Scheduler struct {
 	// PodConditionUpdater is used only in case of scheduling errors. If we succeed
 	// with scheduling, PodScheduled condition will be updated in apiserver in /bind
 	// handler so that binding and setting PodCondition it is atomic.
-	PodConditionUpdater factory.PodConditionUpdater
+	podConditionUpdater PodConditionUpdater
```

Comment:
What is the reason for doing this?

Reply:
`PodConditionUpdater` is only used by the scheduler package. No need to be exported.

Reply:
I prefer to keep it exported. It could be better to examine all the field of the scheduler and unexported them together.

Reply:
I believe unexporting the field is strongly related to removing it from `factory.Config` as it was originally exported to be used by the scheduler. Also, it will simplify the process of unexporting fields later on. I also get your point about exporting the fields altogether, so let's see what others think about it